services/src/integ-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterIntTest.java

```diff
@@ -80,21 +82,10 @@ public class MessageWriterIntTest extends AbstractIntegrationTest {
     @Test
     public void testQueueMessage() throws JMSException, InterruptedException {
         // Get message writer from Spring.
-        final MessageWriter messageWriter =
-                (MessageWriter) this.applicationContext
-                        .getBean("uk.gov.moj.sdt.services.messaging.api.IMessageWriter");
+        final MessageWriter messageWriter = (MessageWriter) this.applicationContext.getBean("MessageWriter");
 
         final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
 
-        // Clear any old messages off the queue.
```

Comment:
No need to do this now because ... ?

Reply:
as we have listener attached already in messageListenerContainerMCol -> ServicesTestConfig, which is already consuming messsages